open_instruct/utils.py

```diff
@@ -639,6 +644,18 @@ class FlatArguments:
         default=3,
         metadata={"help": "How many checkpoints to keep in the output directory. -1 for all."},
     )
+    push_to_hub: bool = True
+    """Whether to upload the saved model to huggingface"""
+    hf_entity: Optional[str] = None
+    """The user or org name of the model repository from the Hugging Face Hub"""
+    hf_repo_id: Optional[str] = None
+    """The id of the saved model in the Hugging Face Hub (can be autoset if not given)"""
+    hf_repo_revision: Optional[str] = None
+    """The revision of the saved model in the Hugging Face Hub (can be autoset if not given)"""
+    hf_repo_url: Optional[str] = None
+    """The url of the saved model in the Hugging Face Hub (will be autoset)"""
+    launch_beaker_eval_jobs: bool = False
+    """Whether to launch beaker evaluation jobs after training"""
```

Comment:
as noted above, it looks like `launch_beaker_eval_jobs` requires `push_to_hub` to be set to true, so we should assert that in the post init.